lib/std/rand.zig

```diff
@@ -15,6 +15,8 @@ const math = std.math;
 const ziggurat = @import("rand/ziggurat.zig");
 const maxInt = std.math.maxInt;
 
+const Dilbert = @import("rand/Dilbert.zig");
```

Comment:
If you name the file with `_test.zig` in the name then it won't get installed as part of the standard library.